test/cuda-arch/base/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(cuda-clang-cpp LANGUAGES CXX)
+
+set(BLT_CXX_STD c++14 CACHE STRING "")
+
+# This test will verify that CUDA compile flags that are configurable by the user
+# (these are the only two, besides the dir for the CUDA toolkit) will correctly be
```

Comment:
> \# This test will verify that CUDA compile flags that are configurable by the user
\# (these are the only two, besides the dir for the CUDA toolkit) will correctly be
\# passed down to downstream projects.

It would be helpful if this comment were more explicit -- what are the two CUDA compile flags that are configurable by the user?